cmd/meroxa/root/endpoints/create_endpoint.go

```diff
@@ -0,0 +1,82 @@
+package endpoints
+
+import (
+	"context"
+
+	"github.com/meroxa/meroxa-go"
+
+	"github.com/meroxa/cli/cmd/meroxa/builder"
+	"github.com/meroxa/cli/log"
+)
+
+type createEndpointClient interface {
+	CreateEndpoint(ctx context.Context, name, protocol, stream string) error
+}
+
+type CreateEndpoint struct {
+	client createEndpointClient
+	logger log.Logger
+
+	args struct {
+		Name string
+	}
+
+	flags struct {
+		Protocol string `long:"protocol" short:"p" usage:"protocol, value can be http or grpc" required:"true"`
+		Stream   string `long:"stream" short:"s" usage:"stream name" required:"true"`
+	}
+}
+
+func (c *CreateEndpoint) Execute(ctx context.Context) error {
+	c.logger.Info(ctx, "Creating endpoint...")
+
+	err := c.client.CreateEndpoint(ctx, c.args.Name, c.flags.Protocol, c.flags.Stream)
+
+	if err != nil {
+		return err
+	}
+
+	c.logger.Info(ctx, "Endpoint successfully created!")
```

Comment:
We don't support JSON output? 😕

Reply:
@lovromazgon nope, unless we fetch for an endpoint just for that reason. I thought about doing that, but I think I'd prefer updating the `createEndpoint` in meroxa-go to return the endpoint created for this.

Reply:
Tracking here https://meroxa.atlassian.net/browse/PLATFORM-571.